### DISCUSSION TITLE
Will operations in CTS like `clock_tree_synthesis` and `repair_timing` change the data paths?

### DISCUSSION THREAD

**Discussion by dearmrlv** [The-OpenROAD-Project/OpenROAD#5716] [Q&A]
A timing path is consisted of the data path and the clock path. Do commands in CTS like `clock_tree_synthesis` and `repair_timing` optimize both the data paths or the clock paths? If I want those commands do optimization on ONLY the clock paths, how should I configure? Thanks for your answers!

**Comment by maliberty** [comment]
repair_timing only works on the data path and there is no option to do otherwise.  repair_clock_nets is probably what you want.  CTS creates the clock tree and does not affect the data path.
